vector/src/main/java/im/vector/app/features/home/room/detail/timeline/action/MessageActionsViewModel.kt

```diff
@@ -450,7 +450,8 @@ class MessageActionsViewModel @AssistedInject constructor(
     private fun canReplyInThread(event: TimelineEvent,
                                  messageContent: MessageContent?,
                                  actionPermissions: ActionPermissions): Boolean {
-        if (!vectorPreferences.areThreadMessagesEnabled()) return false
+/* We let reply in thread visible even if threads are not enabled, with an enhanced flow to attract users */
+//        if (!vectorPreferences.areThreadMessagesEnabled()) return false
```

Comment:
I would prefer to add some indentation here and for the comment above.

Reply:
Done